### ISSUE TITLE
add reference flux/magnitude as an output column

### ISSUE THREAD

**Issue by janvanroestel** [lukeshingles/atlasserver#11]
In the case of variable stars (or long-timescale transients), the source has a non-zero flux in reference images. To reconstruct the lightcurve, the flux in the reference image is required. Also, an update of the reference image can cause discontinuities in lightcurves of (very) long timescale transients.

This can be solved by adding a column with the PSF-flux/mag in the reference image.

**Comment by lukeshingles** [comment]
If you enable 'Use reduced instead of difference images', then you should get back an absolute flux from the input images rather than a difference flux. We should add better documentation for that switch. Does using that option solve your use case?

**Comment by ekhammer** [comment]
It would be helpful to see which reference image (or some identifier for the reference image) was used for each observation, as for some sources I can see that the reference image was updated to one where there is a non-zero flux from the transient itself; like @janvanroestel mentioned, this causes a discontinuity in the light curve that could be fixed if the reference image flux for each observation was known. For my use, I need a difference flux light curve so this would be helpful.

**Comment by lukeshingles** [comment]
I mentioned this to John Tonry (who wrote the script that creates the data files) and it sounded as though this might be added in the future. I'll update this issue when I have more information.

**Comment by lukeshingles** [comment]
We looked into this in some detail and unfortunately, it's not always possible to identify the exact reference template that was used to calculate the difference images going back through the history of the survey. For recent images, new headers were been added to solve this problem. The best we can do is suggest that users request reduced images for comparison around discontinuities and be cautious near MJD 58417 and MJD 58882 (+/- several weeks). We have updated the FAQ with this advice.
